ngraph/python/tests/test_ngraph/test_basic.py

```diff
@@ -417,7 +419,7 @@ def test_runtime_info():
     relu_node.set_friendly_name("testReLU")
     runtime_info_after = relu_node.get_rt_info()
 
-    assert runtime_info == runtime_info_after
+    assert runtime_info["affinity"] == runtime_info_after["affinity"]
```

Comment:
I think this will make the test more readable:
```suggestion
    assert runtime_info_after["affinity"] == "test_affinity"
```